big_tests/tests/privacy_SUITE.erl

```diff
@@ -673,6 +673,7 @@ block_jid_iq(Config) ->
         %% activate it
         Stanza = escalus_stanza:privacy_activate(<<"deny_localhost_iq">>),
         escalus_client:send(Alice, Stanza),
+        escalus:assert(is_iq_result, escalus_client:wait_for_stanza(Alice)),
```

Comment:
it is new, because now iq result (for privacy and blocking iqs) is sent synchronously from the c2s, not routed, so in this case it is guaranteed to go through since it happens before the state is updated.

Reply:
Does it mean that on master we simply don't have the iq result for the iq activating the privacy list?

Reply:
Yes, because it is blocked by the list we've just set.